src/sagemaker/fw_utils.py

```diff
@@ -617,9 +617,9 @@ def warn_if_parameter_server_with_multi_gpu(training_instance_type, distribution
         return
 
     is_multi_gpu_instance = (
-        training_instance_type.split(".")[1].startswith("p")
-        and training_instance_type not in SINGLE_GPU_INSTANCE_TYPES
-    )
+        training_instance_type == "local_gpu"
+        or training_instance_type.split(".")[1].startswith("p")
+    ) and training_instance_type not in SINGLE_GPU_INSTANCE_TYPES
 
     ps_enabled = "parameter_server" in distributions and distributions["parameter_server"].get(
```

Comment:
`"local_gpu"` doesn't support distributed training iirc, so if indeed a parameter server is set, wouldn't it make more sense to assume that the intent is to do distributed training and instead fail that GPU + distributed doesn't work without printing out this ambiguous warning that won't apply?